client/scss/utilities/_root-scope-classes.scss

```diff
@@ -367,3 +367,10 @@
 .overflow-hidden {
   overflow: hidden;
 }
+
+.smooth-font {
+  -webkit-font-smoothing: antialiased;
+  -moz-font-smoothing: antialiased;
+  -o-font-smoothing: antialiased;
```

Comment:
(Sorry for my ignorance) <= when should we use this?

Reply:
Could we apply it to the `font-white` class? Or is it not always?
(given white will always be on a dark background).

Reply:
On closer inspection, this is specific to white Wellcome Bold, so generifying not straightforward unfortunately. Will add a comment.